frontend/app/components/crud-list-item/component.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember'
+
+export default Ember.Component.extend({
+  tagName: '',
+  maskPassword: true,
+  actions: {
+    edit() {
+      this.set('edit', true)
+    },
+    show() {
+      this.set('maskPassword', !this.get('maskPassword'))
+    },
+    save(index, entry) {
+      this.sendAction('save', index, entry)
```

Comment:
AFAIK `sendAction` is not really recommended anymore. Use closure actions instead:
```
this.get('save')(index, entry)
```
Usually, we're also prepending `on-` to the attributes: `{{somecomponent on-save=(action foo) ...}}`